data/SPRING/projects.yaml

```diff
@@ -142,3 +142,31 @@ projects:
         date_updated: 2020-02-21
         maturity: 2
             
+    SwissCovid:
+        name: SwissCovid Exposure Notification
+        description: A privacy preserving exposure notification protocol to inform people about evenutal exposition to the CoVid-19 virus
+        layman_desc: >
+            The C4DT Factory team helped the SwissCovid effort by providing techincal specifications to the already written code.
+            In addition, we code reviewed, and helped Ubique to use github in a more Open Source respecting way.
```

Comment:
→ "we reviewed the code"

Reply:
Oh my - I need to go out more and talk to actual people...